lib/slack/block_kit/element/radio_buttons.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Slack
+  module BlockKit
+    module Element
+      # A radio button group that allows a user to choose one item from a list of possible options.
+      #
+      # https://api.slack.com/reference/messaging/block-elements#radio
+      class RadioButtons
+        TYPE = 'radio_buttons'
+
+        attr_accessor :options, :initial_option, :confirm
+
+        def initialize(action_id:)
+          @action_id = action_id
+
+          yield(self) if block_given?
+        end
+
+        def confirmation_dialog
+          @confirm = Composition::ConfirmationDialog.new
+
+          yield(@confirm) if block_given?
+
+          self
+        end
+
+        def option(value:, text:, initial: false)
```

Comment:
Adding `initial:` as a keyword argument here as the first step towards https://github.com/CGA1123/slack-ruby-block-kit/issues/47